src/modules/Hosts/dynamic-plugin.ts

```diff
@@ -0,0 +1,20 @@
+import { EncodedExtension } from '@openshift/dynamic-plugin-sdk';
+import { RoutePage } from '@openshift-console/dynamic-plugin-sdk';
+import type { ConsolePluginMetadata } from '@openshift-console/dynamic-plugin-sdk-webpack/lib/schema/plugin-package';
+
+export const exposedModules: ConsolePluginMetadata['exposedModules'] = {
+  HostsPage: './modules/Hosts/HostsPageWrapper',
+};
+
+export const extensions: EncodedExtension[] = [
+  {
+    type: 'console.page/route',
+    properties: {
+      component: {
+        $codeRef: 'HostsPage',
+      },
+      path: '/mtv/providers/vsphere/:providerName',
```

Comment:
This looks like it belongs to the providers, does it makes sense to put the hosts in a separate model ?